### DISCUSSION TITLE
[Dawn Breaker Party] Feedback from community

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#2674] [General]
Starting this discussion with the aim of knowing the community's feedback about the event.

To start, I'll give my feedback and hope to stimulate community discussion.

- We should have collision between players.
![image](https://github.com/sunflower-land/sunflower-land/assets/74167741/c2b2c05d-b507-4ac9-acc7-ed9787802358)

- Spawn area with a valid path.
_We should assume that players spawn on valid paths, without other players or objects, within an area that accommodate them._
![map](https://github.com/sunflower-land/sunflower-land/assets/74167741/493e7d69-272b-426c-91d7-0b291dae0b2e)

- Map is too small to accommodate the players it is receiving.
_The entire Dawn break island should be considered as spawn area._

- Player identification.
_The players are very similar we should consider a way to identify them._
![playerName](https://github.com/sunflower-land/sunflower-land/assets/74167741/09ec8209-819b-48f9-a73f-dd3c8d6a860b)

- Shadow on chat text.
_Why not considering a background on this balloon with a higher transparency? Maybe black?_
![image](https://github.com/sunflower-land/sunflower-land/assets/74167741/ce606948-1b8d-4ec3-a614-f3ab865d451f)

- Press ENTER on the keyboard to open and close the chat prompt.
_If chat closed > Press enter to chat > Text Message > If chat open >  Press Enter to send and close_ 
![image](https://github.com/sunflower-land/sunflower-land/assets/74167741/91ed8226-cc28-44be-bbb3-320c0f1f1a7c)
![image](https://github.com/sunflower-land/sunflower-land/assets/74167741/25cebd41-a609-43ab-b97b-dcd8d695edd1)

- Consider the option to moving using ASDW keys.
_It is much more user friendly to play with mouse._

- Joystick on mobile fixed to camera instead of player.
_Much more user friendly, and we can also give the user the option to choose which side (R or L) they want, bringing more accessibility._
![image](https://github.com/sunflower-land/sunflower-land/assets/74167741/81ed31d0-48d9-47a6-8142-4a3f2ca364b4)
![joystick](https://github.com/sunflower-land/sunflower-land/assets/74167741/9e62bf71-3bd5-4ec4-aaad-9655cd873b5c)

- Translating game using i18n and i10n (.mo, .po and .pot).
_Properly translated software not only makes life much easier, but also attracts new users. Sure our community could keep our game properly translated if we had the right tools to do it._

- We seem to have a BUG when users try to enter a new location.
_The user must be disconnected from the current map and all users must receive this information._
![image](https://github.com/sunflower-land/sunflower-land/assets/74167741/76e4cf2a-fa07-45fa-8e74-5d9939120db0)
_After joining new map it seems that there is a BUG when leaving that location, when leaving always I don't go back to the server where I was, I seem to be offline._

- We  also seem to have a BUG on sound config.
_It looks like this map's sound config is not loading the player's current sound config._

Thank you.

**Comment by TheDcol** [comment]
Good feedback mate, I agree with most of these!

They have added farm number id on the bumpkins fyi

**Comment by adamhannigan** [comment]
Thanks for raising these 🙏 

We are doing some critical updates and will include these in there
